resources/definitions/zav_maxpro.def.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name":      "ZAV PRO",
```

Comment:
The [product page](https://www.zav3d.ru/products/38842616) for this printer indicates that the printer's build volume is 300x200x260. But this printer definition allows a height of 270mm instead of 260. Which of the two is right?

Reply:
270 is correct. With new tables there will be 270